src/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+import { DISABLED_ELEMENTS } from "./utils";
+
+export {DISABLED_ELEMENTS}
```

Comment:
I don't really think this file is necessary

Reply:
removed